frontend/src/components/LoginPage/LoginPage.css

```
@@ -0,0 +1,141 @@
+#loginBox {
+    height: 450px;
+    width: 350px;
+    background: #FEFEFE;
+    border: 0.5px solid #C4C4C4;
```

```suggestion
    border: none;
```
I know the Figma shows a 0.5px border, but it doesn't show up in the app as expected, so I just got rid of it.